omod/src/main/java/org/openmrs/module/appointments/web/mapper/AppointmentMapper.java

```diff
@@ -128,8 +128,7 @@ private void mapProvidersForAppointment(Appointment appointment, List<Appointmen
 
         if (existingProviders != null) {
             for (AppointmentProvider appointmentProvider : existingProviders) {
-                boolean exists = newProviders == null ? false :
-                        newProviders.stream().anyMatch(p -> p.getUuid().equals(appointmentProvider.getProvider().getUuid()));
+                boolean exists = newProviders != null && newProviders.stream().anyMatch(p -> p.getUuid().equals(appointmentProvider.getProvider().getUuid()));
```

Comment:
Use proper names instead of `p`. Do it in other places

Reply:
changed.